src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionAddActionTest.java

```diff
@@ -113,8 +115,8 @@ public void testExecuteAndPostProcessMsq() {
         action = getAction(params);
         result = getRedirectResult(action);
 
-        assertEquals(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE + "?courseid=" + instructor1ofCourse1.courseId
-                     + "&fsname=First+feedback+session" + "&user=" + instructor1ofCourse1.googleId + "&error=false",
+        assertEquals(getPageResultDestination(Const.ActionURIs.INSTRUCTOR_FEEDBACK_EDIT_PAGE,
+                instructor1ofCourse1.courseId, "First+feedback+session", instructor1ofCourse1.googleId, false),
                      result.getDestinationWithParams());
```

Comment:
The outer parameter `result.getDestinationWithParams()` should not be further indented than the parameters of `getPageResultDestination`.